packages/trader/src/App/Containers/trade-settings-extensions.tsx

```diff
@@ -19,15 +24,17 @@ const ChartSettingContainer = Loadable({
 //     loading: UILoader,
 // });
 
-const renderItemValue = (props, store) => (
-    <MobxContentProvider store={store}>
+const renderItemValue = <T extends object>(props: T, store: TCoreStores) => (
+    <TraderProviders store={store}>
         <ChartSettingContainer {...props} />
-    </MobxContentProvider>
+    </TraderProviders>
 );
 
-const TradeSettingsExtensions = ({ populateSettingsExtensions, store }) => {
+const TradeSettingsExtensions = observer(({ store }: TTradeSettingsExtensionsProps) => {
+    const { ui } = useStore();
+    const { populateSettingsExtensions } = ui;
     const populateSettings = () => {
-        const menu_items = [
+        const menu_items: Array<TPopulateSettingsExtensionsMenuItem> | null = [
```

Comment:
```suggestion
        const menu_items: Parameters<typeof populateSettingsExtensions>[0] = [
```
@jim-deriv Let's not export `TPopulateSettingsExtensionsMenuItem` from stores and instead use generics 🙏🏻